packages/react/kibana_context/styled/README.mdx

```diff
@@ -0,0 +1,15 @@
+---
+id: react/context/styled
+slug: /react/context/styled
+title: React Context - `styled-components` provider
+description: Before `emotion` was introduced, some components used `styled-components` to easily apply EUI variables and apply style.  This package is for compatibility with those components and should _not_ be used in new code.
+tags: ['shared-ux', 'react', 'context']
+date: 2023-07-25
+---
+
+## Description
+
+Before `emotion` was introduced, some components used `styled-components` to easily apply EUI variables and apply style.  This package is an isolated location for this code and included for compatibility with those components.
```

Comment:
Thanks for clarifying this! I'll update our [EUI FAQ](https://github.com/elastic/eui/discussions/6828#discussioncomment-6076157) to note that we're starting to more actively ask Kibana devs to migrate away from `styled-components`.

Not sure if it's worth documenting; plugins can't use both `styled-components` and `@emotion/react` simultaneously, their babel plugins conflict. We've gotten a few confused questions here at EUI about that.

Reply:
I can include this in a follow-up PR, no problem.

Reply:
Addressed in #162757